apps/web/src/components/PoolCandidatesTable/helpers.tsx

```diff
@@ -206,6 +212,31 @@ export const finalDecisionCell = (
   return <Chip color={color}>{label}</Chip>;
 };
 
+export const publicFacingStatusCell = (
+  submittedAt: PoolCandidate["submittedAt"],
+  closingDate: Pool["closingDate"],
+  removedAt: PoolCandidate["removedAt"],
+  finalDecisionAt: PoolCandidate["finalDecisionAt"],
+  finalDecision: Maybe<FinalDecision> | undefined,
+  areaOfSelection: Maybe<PoolAreaOfSelection> | undefined,
+  assessmentStatus: PoolCandidate["assessmentStatus"],
+  screeningQuestions: Pool["screeningQuestions"],
```

Comment:
Think the linter will be happy by just changing that to `Pool["screeningQuestionsCount"]`

Reply:
[775bf01](https://github.com/GCTC-NTGC/gc-digital-talent/pull/14112/commits/775bf012bc389d8a9a5a83c97c26ae5518907acd)